### PR TITLE
Feat: back button on auth pages

### PR DESCRIPTION
**Pull request by Abots-jzm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "styled-components": "^5.3.6"
   },
   "scripts": {
-    "start": "PORT=3666 react-scripts start",
```
Comment (AlexChika) [inline package.json:21]: revert this change. add the port before commiting.

Add back the port. You may remove the port to run on local

**Review by AlexChika — COMMENTED** [review]
Please make the change commit and push.

File: package.json (hunk 1 above)
Comment (Abots-jzm) [reply package.json]: done
